$OpenBSD: patch-jbsrc_jb_c,v 1.4 2011/04/16 09:13:08 ajacoutot Exp $

- fix gtk-update-icon-cache line.
- fix build with GMime 2.4.

--- jbsrc/jb.c.orig	Thu May 22 17:47:04 2008
+++ jbsrc/jb.c	Sat Sep 11 09:54:45 2010
@@ -166,7 +166,7 @@ jb_package_configure (void)
   jb_require_packages("GNOME", "gnome", "glib-2.0 >= 2.14 gthread-2.0 gconf-2.0 >= 2.4.0 gtk+-2.0 >= 2.12 libgnomeui-2.0 >= 2.14.0 gnome-vfs-2.0 libglade-2.0 libxml-2.0 libnotify >= 0.4.1");
   jb_require_packages("D-Bus", "dbus", "dbus-glib-1");
 
-  jb_check_packages_for_options("GMime", "gmime", "gmime-2.0 >= 2.2.7",
+  jb_check_packages_for_options("GMime", "gmime", "gmime-2.4 >= 2.4.0",
 				"hotmail",
 				"imap",
 				"maildir",
@@ -365,8 +365,6 @@ jb_package_add_resources (void)
     jb_group_add_data_file(group, "hotmail.png", "$pkgdatadir");
 
   rule = jb_rule_new();
-  jb_rule_set_install_message(rule, "updating the GTK+ icon cache");
-  jb_rule_add_install_command(rule, "-gtk-update-icon-cache -f -t $datadir/icons/hicolor");
   jb_group_add_resource(group, JB_GROUP_RESOURCE(rule));
 
   jb_group_add(group);
